pyproject.toml

```
@@ -1,8 +1,12 @@
 [tool.poetry]
 name = "django-gov-notify"
 version = "0.2.0"
-description = ""
+description = "A GOV.UK Notify flavoured Django email backend"
 authors = ["Nick Smith <nick.smith@torchbox.com>"]
+readme = "README.md"
+homepage = "https://github.com/nimasmi/django-gov-notify"
+repository = "https://github.com/nimasmi/django-gov-notify"
+keywords = ["email", "Django"]
```

How about expanding this a bit to include classifiers and license?

```suggestion
keywords = ["email", "Django"]
classifiers = [
    "Development Status :: 4 - Beta",
    "Environment :: Web Environment",
    "Intended Audience :: Developers",
    "Operating System :: OS Independent",
    "Programming Language :: Python",
    "Programming Language :: Python :: 3",
    "Topic :: Communications :: Email",
    "Framework :: Django",
    "Framework :: Django :: 2.2",
    "Framework :: Django :: 3.1",
    "Framework :: Django :: 3.2",
]
license = BSD-2-Clause"
```